cli/agent_test.go

```diff
@@ -44,17 +44,17 @@ func TestWorkspaceAgent(t *testing.T) {
 			"--log-dir", logDir,
 		)
 
-		pty := ptytest.New(t).Attach(inv)
```

Comment:
attaching a PTY to agent invocation results in logging twice to the test log: once via our logger and a second time via stderr.  I modified the test to not need the PTY at all.